controller/control.go

```diff
@@ -966,12 +969,14 @@ func (c *Controller) WriteAt(b []byte, off int64) (int, error) {
 
 func (c *Controller) Sync() (int, error) {
 	c.Lock()
-	if c.ReadOnly == true {
+	if c.ReadOnly && !c.logged {
+		c.logged = true
 		err := fmt.Errorf("Mode: ReadOnly")
 		c.Unlock()
 		time.Sleep(1 * time.Second)
 		return -1, err
 	}
+	c.logged = false
```

Comment:
Let's also put a log saying, volume started serving IOs. Even though we have the log times when the volume became RO and RW, but we would also like to know the timestamps when the first time IOs were unserved and when the first time IOs were being served again.